static/app/views/performance/transactionSummary/transactionSpans/spanSummary/content.tsx

```diff
@@ -0,0 +1,137 @@
+import {getSelectedProjectPlatforms} from 'sentry/views/performance/utils';
+import {useSpanMetrics} from 'sentry/views/starfish/queries/useDiscover';
+import type {SpanMetricsQueryFilters} from 'sentry/views/starfish/types';
+
+import Tab from '../../tabs';
+
+import SpanSummaryControls from './spanSummaryControls';
+import SpanSummaryHeader from './spanSummaryHeader';
+
+type Props = {
+  eventView: EventView;
+  location: Location;
+  organization: Organization;
+  project: Project | undefined;
+  spanSlug: SpanSlug;
+  transactionName: string;
+};
+
+export default function SpanSummary(props: Props) {
+  const {location, organization, eventView, project, transactionName, spanSlug} = props;
+
+  // customize the route analytics event we send
+  useRouteAnalyticsEventNames(
+    'performance_views.span_summary.view',
+    'Performance Views: Span Summary page viewed'
+  );
+  useRouteAnalyticsParams({
+    project_platforms: project ? getSelectedProjectPlatforms(location, [project]) : '',
+  });
+
+  return (
+    <Fragment>
+      <Layout.Header>
+        <Layout.HeaderContent>
+          <Breadcrumb
+            organization={organization}
+            location={location}
+            transaction={{
+              project: project?.id ?? '',
+              name: transactionName,
+            }}
+            tab={Tab.SPANS}
+            spanSlug={spanSlug}
+          />
+          <Layout.Title>
+            {project && (
+              <IdBadge
+                project={project}
+                avatarSize={28}
+                hideName
+                avatarProps={{hasTooltip: true, tooltip: project.slug}}
+              />
+            )}
+            {transactionName}
+          </Layout.Title>
+        </Layout.HeaderContent>
+      </Layout.Header>
+      <Layout.Body>
+        <Layout.Main fullWidth>
+          <SpanSummaryContent
+            location={location}
+            organization={organization}
+            project={project}
+            eventView={eventView}
+            spanSlug={spanSlug}
+            transactionName={transactionName}
+          />
+        </Layout.Main>
+      </Layout.Body>
+    </Fragment>
+  );
+}
+
+type ContentProps = {
+  eventView: EventView;
+  location: Location;
+  organization: Organization;
+  project: Project | undefined;
+  spanSlug: SpanSlug;
+  transactionName: string;
+};
+
+function SpanSummaryContent(props: ContentProps) {
+  const {transactionName, project} = props;
+
+  const {spanSlug: spanParam} = useParams();
+  const [spanOp, groupId] = spanParam.split(':');
+
+  const filters: SpanMetricsQueryFilters = {
+    'span.group': groupId,
+    'span.op': spanOp,
+    transaction: transactionName,
+  };
+
+  const {data: spanHeaderData} = useSpanMetrics({
+    search: MutableSearch.fromQueryObject(filters),
+    fields: ['span.description', 'avg(span.self_time)', 'sum(span.self_time)', 'count()'],
```

Comment:
```suggestion
    fields: ['span.description', 'avg(span.duration)', 'sum(span.self_time)', 'count()'],
```

Reply:
Yes, let's  make sure we use duration for this page